src/parser.rs

```diff
@@ -174,8 +174,13 @@ fn parse_hunk_start(line: &str) -> Result<Option<(u64, u64)>> {
 
         let hunk_start_line_right: u64 = captures
             .name("rstart")
-            .unwrap()
-            .as_str()
+            .map(|s| s.as_str()).unwrap_or_else(||
+                if hunk_start_line_left == 0 {
+                    "0"
+                } else {
+                    unreachable!("")
```

Comment:
Maybe be loud and panic here in case it does somehow happen?

Reply:
Unreachable also panics, but the error message has improvement potential. Will address